include/exadg/solvers_and_preconditioners/utilities/petsc_operation.h

```diff
@@ -28,38 +28,37 @@ namespace ExaDG
 {
 using namespace dealii;
 
+#ifdef DEAL_II_WITH_PETSC
+/*
+ * A typedef to make use of PETSc data structure clearer
+ */
+typedef Vec VectorTypePETSc;
+
 /*
  *  This function wraps the copy of a PETSc object (sparse matrix,
- *  preconditioner) with a dealii::LinearAlgebra::distributed::Vector.
+ *  preconditioner) with a dealii::LinearAlgebra::distributed::Vector, taking
+ *  pre-allocated PETSc vector objects (with struct name `Vec) for the
```

Comment:
`Vec -> adjust comment and/or incorporate new typedef.

Reply:
See 644e7104.